third_party/terraform/utils/utils.go.erb

```diff
@@ -60,6 +60,13 @@ func getProject(d TerraformResourceData, config *Config) (string, error) {
 	return getProjectFromSchema("project", d, config)
 }
 
+// getBillingProject reads the "billing_project" field from the given resource data and falls
+// back to the provider's value if not given. If the provider's value is not
```

Comment:
`"If the provider's value is not given, an error is returned."` => `"If no value is found, an error is returned."`?

Since this field is optional, I think it's worth breaking away from the wording of the required fields.